src/startUp.ts

```diff
@@ -26,12 +26,19 @@ const checkArgs = (args: IArgConfig) => {
         console.error("Options 'rate-limit-global-windowms' and 'rate-limit-global-max' must be provided together.");
         process.exit(1);
     }
+
+    if (commandLineConfig.loglevel && !Object.values(LogLevel).includes(commandLineConfig.loglevel)) {
```

Comment:
This should use tryParse

Reply:
it gets a bit awkward, though, as tryParse returns a LogLevelInfo instance. Then I should generalize setLogLevel to work with both LogLevel or LogLevelInfo.

Reply:
just use the property on LogLevelInfo

logLevelInfo.Level

Reply:
Changed with TryParse, and also changed the signature of setLogLevel (see other comment below).